content/guides/core/automations/create-automations/webhook.md

```diff
@@ -0,0 +1,302 @@
+
+{{% /tab %}}
+{{% tab "Project" %}}
+A W&B admin can create automations in a project.
+
+1. Log in to W&B and go to the project page.
+1. In the sidebar, click **Automations**.
+1. Click **Create automation**.
+1. Choose the [**Event**]({{< relref "/guides/core/automations/automation-events.md" >}}) to watch for.
+
+    1. Fill in any additional fields that appear, which depend upon the event. For example, if you select **An artifact alias is added**, you must specify the **Alias regex**.
+
+    1. Optionally specify a collection filter. Otherwise, the automation is applied to all collections in the project, including those added in the future.
+    
+    Click **Next step**.
+1. Select the team that owns the [webhook integration]({{< relref "#add-a-webhook-integration" >}}).
+1. Set **Action type** to **Webhooks**. then select the [webhook integration]({{< relref "#add-a-webhook-integration" >}}) to use. 
+1. If your webhook requires a payload, construct it and paste it into the **Payload** field. If you configured an access token for the webhook, you can access the token in the `${ACCESS_TOKEN}` [payload variable]({{< relref "#payload-variables" >}}). If you configured a secret for the webhook, you can access it in the payload by prefixing its name with `$`. Your webhook's requirements are determined by the webhook's service.
+1. Click **Next step**.
+1. Provide a name for the automation. Optionally, provide a description. Click **Create automation**.
+
+{{% /tab %}}
+{{< /tabpane >}}
+
+
+## View and manage automations
+{{< tabpane text=true >}}
+{{% tab "Registry" %}}
+
+- Manage the registry's automations from the registry's **Automations** tab.
+- Manage a collection's automations from the **Automations** section of the collection's details page.
+
+From either of these pages, a Registry admin can manage existing automations:
+- To view an automation's details, click its name.
+- To edit an automation, click its action `...` menu, then click **Edit automation**.
+- To delete an automation, click its action `...` menu, then click **Delete automation**. Confiruation is required.
+
+
+{{% /tab %}}
+{{% tab "Project" %}}
+A W&B admin can view and manage a project's automations from the project's **Automations** tab.
+
+- To view an automation's details, click its name.
+- To edit an automation, click its action `...` menu, then click **Edit automation**.
+- To delete an automation, click its action `...` menu, then click **Delete automation**. Confiruation is required.
+{{% /tab %}}
+{{< /tabpane >}}
+
+## Payload reference
+Use these sections to construct your webhoook's payload. For details about testing your webhook and its payload, refer to [Troubleshoot your webhook]({{< relref "#troubleshoot-your-webhook" >}}).
+
+### Payload variables
+This section describes the variables you can use to construct your webhook's payload.
+
+| Variable | Details |
```

Comment:
Is there any new content here? If there's not, what is the value in adding this here? It seems like if we want to list out the payload variables, we should have some additional information to provide users. Otherwise it's just content that gets out of date and has to be updated, with no added value.

Reply:
This is not new, but was previously in a different page (the Model Registry page this PR deletes). I've added some variables that were previously missing. When @noaleetz thought this was missing due to the Github UI, she asked for it to be reinstated!